emacs/merlin.el

```diff
@@ -1637,7 +1637,7 @@ Empty string defaults to jumping to all these."
   (merlin--goto-file-and-point (merlin-call-jump target)))
 
 (defun merlin-call-phrase (target)
-  "Move to next phrase (TARGET = 'next) or previous phrase (TARGET = 'prev)"
+  "Move to next phrase (TARGET = `next') or previous phrase (TARGET = `prev')."
```

Comment:
I wouldn't be using quote here at all, as I think they actually make things more confusing.